### ISSUE TITLE
Color fringing on text

### ISSUE THREAD

**Issue by 9ary** [anegostudios/VintageStory-Issues#7696]
### Game Version

v1.21.6

### Platform

Linux

### Modded

None

### SP/MP

None

### Description

GUI text has very noticeable color fringing, typical of subpixel rendering. This is very uncomfortable on the eyes; I usually disable subpixel rendering everywhere because of this. I am running the game at 1080p on a 4k display so any attempt at subpixel rendering would be wrong anyway.

I am using NixOS 25.11 with the game as packaged in Nixpkgs.

This doesn't seem like a universal issue, but I could find someone else complaining about this on the forum over a year ago: https://www.vintagestory.at/forums/topic/11935-struggling-with-gui-text/

### How to reproduce

1. Open the game
2. Observe incorrect text rendering

### Screenshots

<img width="3824" height="2064" alt="Image" src="https://github.com/user-attachments/assets/1b2493c6-e283-476d-96d3-685720fa28db" />

### Logs (Required)

[client-crash.log](https://github.com/user-attachments/files/24272620/client-crash.log)
[client-audit.log](https://github.com/user-attachments/files/24272622/client-audit.log)
[server-worldgen.log](https://github.com/user-attachments/files/24272623/server-worldgen.log)
[server-build.log](https://github.com/user-attachments/files/24272618/server-build.log)
[server-chat.log](https://github.com/user-attachments/files/24272626/server-chat.log)
[server-audit.log](https://github.com/user-attachments/files/24272619/server-audit.log)
[server-debug.log](https://github.com/user-attachments/files/24272625/server-debug.log)
[client-debug.log](https://github.com/user-attachments/files/24272627/client-debug.log)
[client-chat.log](https://github.com/user-attachments/files/24272624/client-chat.log)
[server-main.log](https://github.com/user-attachments/files/24272621/server-main.log)
[client-main.log](https://github.com/user-attachments/files/24272617/client-main.log)

**Comment by 9ary** [comment]
This also reproduces on Windows, though it is far more subtle there:
<img width="1026" height="141" alt="Image" src="https://github.com/user-attachments/assets/fb8f53ef-694d-4276-8d71-86f94a61ce19" />

This persists after setting the AA mode to grayscale with https://github.com/bp2008/BetterClearTypeTuner.

Worse yet, Vintage Story applies subpixel AA to text on 3D surfaces, such as signs:
<img width="1865" height="1010" alt="Image" src="https://github.com/user-attachments/assets/c5eef653-abcf-432a-910a-bb71e6e64c46" />
It's very faint because the text is black on dark, but there is a visible blue shift on the left border of the glyphs, and red on the right.
